### ISSUE TITLE
Grid header always show In Electron app when I set property "header" with false

### ISSUE THREAD

**Issue by k3vinlusec** [vitmalina/w2ui#1777]
Hi, 

I'm developing an electron app on macOS, and use w2ui to develop UI.  The following is the html page. I tested it in an electron app with w2ui v1.5, but it always show the grid header even I set property "header" with false. But It worked well when I test the page in latest version Chrome.

Note: The electron version is 2.0.0. I also tested the page  with w2ui v1.3 and it works well.

<!DOCTYPE html>
<html>
<head>
    <title>testDemo</title>
   
    
    <script>if (typeof module === 'object') {window.module = module; module = undefined;}</script>
    
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    
    <script>if (window.module) module = window.module;</script>
     
    
   <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.0/jquery.min.js"></script>
    <script type="text/javascript" src="http://rawgit.com/vitmalina/w2ui/master/dist/w2ui.min.js"></script>
    <link rel="stylesheet" type="text/css" href="http://rawgit.com/vitmalina/w2ui/master/dist/w2ui.min.css" />
    
</head>
<body>
   

<div id="toolbar" style="padding: 4px; border: 1px solid #dfdfdf; border-radius: 3px; "></div>
<div id="layout" style="width: 100%; height: 600px;"></div>

<script type="text/javascript">

$(function () {

    $('#toolbar').w2toolbar({
        name: 'toolbar',
        items: [
            { 
                type: "button", id: "start", text: "Start", //icon: "fa-play"
            },
            {
                type: "button", id: "stop", text: "Stop", disabled: true//icon: "fa-stop"
            },
            { type: 'break' },
            {
                type: "check", id: "procexec", text: "Process Exec", img: 'icon-page' 
            },
            {
                type: "check", id: "network", text: "Network", img: 'icon-page' 
            },
            {
                type: "menu-check", id: "file", text: "File",img: 'icon-page',
                onRefresh: function (event) {
                    event.item.count = event.item.selected.length;
                },
                items: [
                    { id: 'fid1', text: 'File Open',  },
                    { id: 'fid2', text: 'File Read',  },
                    { id: 'fid3', text: 'File Write',  },
                    { id: 'fid4', text: 'File Rename',  },
                    { id: 'fid5', text: 'File Delete',  }] 
            },
            {
                type: "check", id: "dylibload", text: "Dylib Load",img: 'icon-page', //checked: true
            },

        ],
       
    });

    var pstyle = 'border: 1px solid #dfdfdf; padding: 1px; ';
    $('#layout').w2layout({
        name: 'layout',
        padding: 1,
        panels: [
           
                
            //{ type: 'left', size: 200, resizable: true, style: pstyle, content: 'left' },
            { type: 'main', size: 200, style: pstyle, content: 'main' },
            { type: 'preview', size: '30%', resizable: true, style: pstyle, content: '<div style="text-align: center">JSON Data</div>'},
            //{ type: 'right', size: 200, resizable: true, style: pstyle, content: 'right' }
        ]
    });

    $().w2grid({ 
		name	: 'grid', 
        
		show: { 
            header: false,
			toolbar: true,
            footer: true,
            lineNumbers: true,
            toolbarSave    : true, 
		},
		columns: [	
            //{ field: 'recid', caption: 'RecID', size: '60px', sortable: true, searchable: 'int' },			
			{ field: 'time', caption: 'Timestamp', size: '100px', sortable: true, resizable: true, searchable: true },
			{ field: 'eventType', caption: 'Event type', size: '200px', sortable: true, resizable: true, searchable: true },
			{ field: 'process', caption: 'Process', size: '200px', sortable: true, resizable: true, searchable: true },
            { field: 'message', caption: 'Message', size: '100%', resizable: true, sortable: true, searchable: true },
            { field: 'extra', caption: 'Extra', size: '200px', sortable: true, resizable: true}
        ],
        records: [    
        ],

        onClick: function (event){
            var record = this.get(event.recid);
            console.log(record);
            w2ui['layout'].content('preview', '<pre>'+JSON.stringify(record,undefined, 2)+'</pre>');
        }
	});
    w2ui['layout'].content('main', w2ui['grid']);
    
    });

</script>

</body>
</html>

Could you help me check it? Thanks in advance!

Kevin

**Comment by mpf82** [comment]
Can you please provide a Fiddle/Bin/Pen to demonstrate the issue?

Personally, I'm not working with Electron, so help from the community is appreciated if this should happen to be an Electron-only issue.

**Comment by mpf82** [comment]
Closing issue: no response
